NYPL/fedora_ingest_rails
# [DR-2557] Adding script to remove empty containers and collections.

## Jira Tickets ##
[Run cleanup script for bad records in RepoAPI index](https://jira.nypl.org/browse/DR-2557)

## Things Done ##
- Created new script in one-offs folder to delete empty containers and collections from solr. 

## How to Test ##
- Spot check the uuids listed in the csv files. These are what would be deleted if there script were run. 
- Run the script against the test solr of your choice. E.g., for dev: `bundle exec ruby ./scripts/one-offs/DR-2557/delete-empty-records-DR-2557.rb http://10.225.133.217:8983/solr/repoapi` 
- Optionally, add a second argument to run only in test mode, which doesn't commit any changes to solr but generates a list of what would be deleted. E.g., for dev: `bundle exec ruby ./scripts/one-offs/DR-2557/delete-empty-records-DR-2557.rb http://10.225.133.217:8983/solr/repoapi test` 
- It should run for an hour or two and create a csv listing uuids deleted.
